### PR TITLE
Add vendor Marvell

### PR DESCRIPTION
**Pull request by splaspood**
This PR adds a vendor name mapping for Marvell and one of their product ids for identification.

### CODE REVIEW

File: constants.go [hunk 1]
```diff
@@ -134,6 +137,8 @@ func VendorFromString(s string) string {
 		return VendorMellanox
 	case strings.Contains(s, "infineon"):
 		return VendorInfineon
+	case strings.Contains(s, "1b4b-9230"):
```
Comment (joelrebel) [inline constants.go]: It would be worth defining this Marvel PCI ID as a constant and referencing it here,
took me a bit to figure that was a PCI ID :)

**Review by joelrebel — COMMENTED** [review]
A minor suggestion added, apart from that LGTM!

File: constants.go (hunk 1 above)
Comment (splaspood) [reply constants.go]: I agree and made some changes that hopefully clear this up.

**Review by joelrebel — APPROVED** [review]
LGTM, thanks!

**Pull request merged**